spyder_line_profiler/spyder/widgets.py

```diff
@@ -100,123 +100,214 @@ def natural_sort_key(key):
                      for i, e in enumerate(parts))
 
 
-def is_lineprofiler_installed():
-    """
-    Checks if the program and the library for line_profiler is installed.
-    """
-    return (programs.is_module_installed('line_profiler')
-            and programs.find_program('kernprof') is not None)
+class SpyderLineProfilerWidgetActions:
+    # Triggers
+    Browse = 'browse_action'
+    Clear = 'clear_action'
+    Collapse = 'collapse_action'
+    Expand = 'expand_action'
+    LoadData = 'load_data_action'
+    Run = 'run_action'
+    SaveData = 'save_data_action'
+    ShowOutput = 'show_output_action'
+    Stop = 'stop_action'
 
 
-class LineProfilerWidget(QWidget):
-    """
-    Line profiler widget.
-    """
+class SpyderLineProfilerWidgetMainToolbarSections:
+    Main = 'main_section'
+    ExpandCollaps = 'expand_collaps_section'
+    ShowOutput = 'show_output_section'
+
+
+class SpyderLineProfilerWidgetToolbars:
+    Information = 'information_toolbar'
+
+
+class SpyderLineProfilerWidgetMainToolbarItems:
+    FileCombo = 'file_combo'
+
+
+class SpyderLineProfilerWidgetInformationToolbarSections:
+    Main = 'main_section'
+
+
+class SpyderLineProfilerWidgetInformationToolbarItems:
+    Stretcher1 = 'stretcher_1'
+    Stretcher2 = 'stretcher_2'
+    DateLabel = 'date_label'  
```

Comment:
Remove trailing whitespace:
```suggestion
    DateLabel = 'date_label'
```